Makefile

```diff
@@ -1,11 +1,11 @@
 
 BUILD_DIR=build
-COVERAGE_DIR=${BUILD_DIR}/coverage
+COVERAGE_DIR=$(BUILD_DIR)/coverage
```

Comment:
Is there a specific reason for changing `{´ to `(`. Nothing agains it more curious as it does not really belong to this PR.